dataikuapi/dssclient.py

```diff
@@ -760,18 +760,45 @@ def get_global_usage_summary(self, with_per_project=False):
     ########################################################
 
     def get_variables(self):
+        """
+        Deprecated. Use get_global_variables
+        """
+        warnings.warn("get_variables is deprecated, please use get_global_variables", DeprecationWarning)
+        return self.get_global_variables()
+
+    def get_global_variables(self):
         """
         Get the DSS instance's variables, as a Python dictionary
 
         This call requires an API key with admin rights
-        
-        :returns: a Python dictionary of the instance-level variables
+
+        :returns: A :class:`dataikuapi.dss.admin.DSSInstanceVariables` handle
+        """
+        variables = self._perform_json("GET", "/admin/variables/")
+        return DSSInstanceVariables(self, variables)
+
+    def get_resolved_variables(self, project_key=None, typed=False):
```

Comment:
Even though the CH mentions `DSSClient.get_resolved_variables()`, I think this belongs in `DSSProject`, alongside the existing `DSSProject.get_variables()`.